tools-image/build-arm-image.sh

```diff
@@ -71,7 +71,17 @@ cleanup() {
   if [ -n "$oem" ]; then
     umount $oem || true
   fi
-  losetup -D || true
+
+  if [ "$disable_lvm" == "false" ]; then
+    lvremove --yes KairosVG
+  fi
+
+  for LOOPS in $(losetup -a | awk -F':' {'print $1'} | awk -F'/' {'print $3'}); do
```

Comment:
umm good point, there is no real way of discerning pre-existing devices vs created devices I guess?

I have zero loop devices so no idea. But this fixes all my issues with building more than once :D

Also notice that it already did a `losetup -D || true` before which means `Detach all associated loop devices` so it would brute-force its way trougth all devices, this just removes any dms in the system and removes the lvm group.

I guess we could be more specific, get the next free loop device at the start and then use it and clean that one specifically?

Reply:
oh wait, we already have it around, nice, Ill rework this!